lib/slime/compiler.ex

```diff
@@ -33,42 +41,54 @@ defmodule Slime.Compiler do
     ~s( #{to_string(name)}="#{value}")
   end
 
+
   defp render_branch(%{type: :doctype, content: text}), do: text
-  defp render_branch(%{type: :text, content: text}), do: text
+  defp render_branch(%{type: :text, content: text}),    do: text
   defp render_branch(%{} = branch) do
-    opening = branch.attributes
-              |> Enum.map(fn {k, v} -> render_attribute(k, v) end)
-              |> Enum.join
-              |> open(branch)
-
+    opening =
+      branch.attributes
+      |> Enum.map(fn {k, v} -> render_attribute(k, v) end)
+      |> Enum.join
+      |> open(branch)
     closing = close(branch)
     opening <> compile(branch.children) <> closing
   end
 
+
   defp open(_, %{type: :eex, content: code, attributes: attrs}) do
     inline = if attrs[:inline], do: "=", else: ""
     "<%#{inline} #{code} %>\r"
   end
-
-  defp open(_, %{type: :html_comment}), do: "<!--"
-  defp open(_, %{type: :ie_comment, content: conditions}), do: "<!--[#{conditions}]>"
+  defp open(_, %{type: :html_comment}) do
+    "<!--"
```

Comment:
Nope, I just missed them, this PR is a bit too big to keep in your head at once. I'll change the others too.